src/node.cc

```diff
@@ -1546,11 +1557,28 @@ static void GetActiveRequests(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   Local<Array> ary = Array::New(args.GetIsolate());
-  int i = 0;
+  Local<Context> ctx = env->context();
+  Local<Function> fn = env->add_properties_by_index_function();
+  Local<Value> argv[5];
+  size_t i = 0;
+
+  for (auto w : *env->req_wrap_queue()) {
+    if (w->persistent().IsEmpty() == false) {
+      argv[i++ % 5] = w->object();
+      if ((i % 5) == 0) {
+        HandleScope scope(env->isolate());
+        fn->Call(ctx, ary, 5, argv).ToLocalChecked();
+        for (size_t j = 0; j < 5; j++) {
+          argv[j] = Local<Value>();
+        }
```

Comment:
In this particular case `auto&` and `auto&&` are more or less interchangeable so I guess it's alright to appease cpplint.

If the type was non-copyable but movable, I'd be more inclined to shut up cpplint with a NOLINT because `auto&&` would hint to the reader that it's okay to move out the value when it's not.